### DISCUSSION TITLE
请问下在ubuntu上使用apt-get install fonts-smiley-sans安装出来的字体，是个这个项目开发的字体么？

### DISCUSSION THREAD

**Discussion by garyyang85** [atelier-anchor/smiley-sans#145] [Q&A]
请问下在ubuntu上使用apt-get install fonts-smiley-sans安装出来的字体，是个这个项目开发的字体么？ 是可以免费商用的么？感谢！

**Comment by stone-zeng** [comment]
根据 https://packages.ubuntu.com/noble/fonts-smiley-sans 的信息，确实和本项目是一致的，但检查后发现提供的字体文件和 [Releases](https://github.com/atelier-anchor/smiley-sans/releases) 中的并不完全一致，具体情况有待进一步核实。
